pkg/otlp/metrics/metrics_translator.go

```diff
@@ -802,7 +802,12 @@ func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consume
 						}
 					}
 				}
-				if t.cfg.withRemapping {
+				if t.cfg.withRenaming {
```

Comment:
I am not sure if this change is sufficient. We also add the `otel` prefix to some kafka metrics. For example: https://github.com/DataDog/opentelemetry-mapping-go/blob/main/pkg/otlp/metrics/kafka_mapping.go#L26. 

Also we will need to convert the `-` character in otel metrics to `_`. I am not sure if normalization is already handled by the Agent or if we need to make this conversion in this library.